lighthouse-core/report/html/report-styles.css

```diff
@@ -792,7 +792,32 @@
   background: var(--color-fail);
 }
 
-
+.icon-button {
+  display: inline-flex;
+  align-items: center;
+  border: solid 2px grey;
+  height: 50px;
+  cursor: pointer;
+  box-shadow: 1px 1px 3px #ccc;
+}
+.icon-button:hover {
+  background-color: var(--color-gray-200);
```

Comment:
this is correct for light mode, but in dark mode (with the filter invert), this is no bueno.

`#BDBDBD` aka `--color-gray-400` is what we use in the dropdown that inverts to the right shade.

Reply:
this filter invert made using colors for other parts of the element hot trash, so I did a lil refactor of the icons